components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -41,106 +47,232 @@ For deployments that are expected to see production-scale workload, we recommend
 * 16 vCPUs.
 
 Roughly 80 GB of the disk space is designated for Chef Automate; the rest is used for
-Habitat Builder and the artifacts it stores. The current implementation uses Minio for
-Habitat artifact storage; we do not support using Artifactory for artifact storage.
+Chef Habitat Builder and the artifacts it stores. The current implementation uses Minio for
+Chef Habitat artifact storage; we do not support using Artifactory for artifact storage.
 
-#### Operating system
-Chef Automate and Habitat Builder require
+### Operating System
+
+Chef Automate and Chef Habitat Builder require:
 
 * a Linux kernel of version 3.2 or greater
 * `systemd` as the init system
 * `useradd`
 * `curl` or `wget`
 * The shell that starts Automate should have a max open files setting of at least 65535
+* Run the installation and bootstrapping procedures as the superuser or use `` at the start of each command.
```

Comment:
@kagarmoe Is there a command missing for ` `` `?  Asking as that throws off the formatting for this document.